build/common.props

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <!-- other options -->
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <Company>Microsoft Corporation.</Company>
+        <Product>Microsoft .NET</Product>
+
+        <!-- pack options -->
+        <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Suggestion: to improve scriptable updates, I decided to create a top-level file called `version.props` for these settings.